drivers/sensor/bme680/bme680.c

```diff
@@ -342,11 +350,28 @@ static int bme680_read_compensation(const struct device *dev)
 	return 0;
 }
 
-static int bme680_chip_init(const struct device *dev)
+static int bme680_init(const struct device *dev)
 {
 	struct bme680_data *data = dev->data;
 	int err;
 
+	err = bme680_bus_check(dev);
+	if (err < 0) {
+		LOG_ERR("Bus not ready for '%s'", dev->name);
+		return err;
+	}
+
+	/*
+	 * When using both (I2C and SPI) sensors, this read operation will also
+	 * be executed by the I2C device.
```

Comment:
Can you clarify why it needs to be in the ifdef, then, if it's safe to do for both bus types?

Reply:
It is safe for both bus types. The idea was to save a little bit of code/time on case only i2c is enabled.
I'll edit the comment to explain this more clearly, ok?